.github/workflows/node.js.yml

```diff
@@ -3,6 +3,9 @@
 
 name: Node.js Build CI
 
+env:

```

Comment:
`env` er dårlig valg av variabelnavn, da det stort sett brukes for å sette `prod` / `dev`.

Er `node_env` bedre? Eventuelt `node_version`?

Reply:
Hehehe, null stress. Lar den kanskje ligge til oktober?🤔